src/Worker/Delivery.php

```diff
@@ -175,10 +176,18 @@ public static function execute($cmd, $item_id, $contact_id)
 
 			case Protocol::DFRN:
 				self::deliverDFRN($cmd, $contact, $owner, $items, $target_item, $public_message, $top_level, $followup);
+
+				if (in_array($cmd, [Delivery::POST, Delivery::COMMENT])) {
```

Comment:
No, but it doesn't seem to be trying again the same task if it wasn't successful anyway like we do with `APDelivery` with an explicit `Worker::defer()`. If we aren't trying again it should be marked as done in the count.

Reply:
We are trying again - but in the queue mechanism.

Reply:
My point exactly. I'm going to leave it as is for now, because this indicator isn't supposed to be exact, and I'm afraid of the complexity of adding success handling which I couldn't find in the current delivery system.

Reply:
Guess why I want to replace the queue mechanism with the mechanism I implemented in the AP delivery process :-)